tests/python/relay/test_dataflow_pattern.py

```diff
@@ -959,4 +984,5 @@ def test_parition_double_batchnorm():
     test_quadruple_partition_dominator()
     test_parition_batchnorm()
     test_parition_double_batchnorm()
+    test_parition_check()
```

Comment:
test_partition

Reply:
Ooo, I should fix a few of those, thanks for noticing!